tasks/install.yml

```diff
@@ -1,9 +1,57 @@
 ---
 # ©Copyright 2015 Hewlett-Packard Development Company, L.P.
 
+- name: Include system dependencies
+  include_vars: "vars{{ ansible_os_family }}.yml"
+
+- name: Include DB dependencies
+  include_vars: "{{ ansible_os_family|lower }}_{{ database_backend }}.yml"
+  when: database_support
+
+- name: Fetch pip getter
```

Comment:
I think that the pip installation is out of scope for this role. It should be done in another role or in the playbook before calling this, pip is just a dependency for this role.